packages/core/src/rules/common/scalar-property-missing-example.ts

```diff
@@ -32,6 +35,10 @@ export const ScalarPropertyMissingExample: Oas3Rule | Oas2Rule = () => {
   };
 };
 
+function isNotDefined(value: unknown): boolean {
```

Comment:
can you move it to some utils? We have somewhere validateDefinedAndNotEmpty

Reply:
Maybe drop this function and simply compare values inline? It doesn't bring much sense to me.

Reply:
validateDefinedAndNotEmpty won't work here. 
I just got rid of the func at all.